quinn-proto/src/token.rs

```diff
@@ -140,27 +146,78 @@ mod test {
     #[test]
     fn token_sanity() {
         use super::*;
-        use crate::MAX_CID_SIZE;
+        use crate::{crypto, MAX_CID_SIZE};
         use rand::RngCore;
-        use ring::hmac;
         use std::{
             net::Ipv6Addr,
             time::{Duration, UNIX_EPOCH},
         };
 
-        let mut key = [0; 64];
-        rand::thread_rng().fill_bytes(&mut key);
-        let key = <hmac::Key as HmacKey>::new(&key).unwrap();
+        let mut master_key = [0; 64];
+        rand::thread_rng().fill_bytes(&mut master_key);
+
+        let mut random_bytes = [0; 32];
+        rand::thread_rng().fill_bytes(&mut random_bytes);
+
+        let rng = &mut rand::thread_rng();
```

Comment:
If we're going to introduce a variable for this, might as well use it in place of the repeated `&mut rand::thread_rng()` below.